### DISCUSSION TITLE
Multiple pages rendering at the same time

### DISCUSSION THREAD

**Discussion by angeligareta** [vercel/next.js#13688] [Help]
Hello, 

I am having an issue while rendering the home page of my webapp. My page structure is:
- index.tsx
- [business]/index.tsx

Can I not indicate somehow that business must be different from **undefined**?

Thanks in advance

**Comment by jamesmosier** [comment]
Hi @angeligareta! Do you happen to have a repository that shows the issue you are encountering?

I tested with this page structure:

```
pages/
  index.tsx
  [business]/
    index.tsx
```

and by going to `http://localhost:3000` I was taken to the `index.tsx` and `http://localhost:3000/foo` I was taken to `[business]/index.tsx`

**Reply by angeligareta** [reply]
Hi James,

Thanks for the answer. I do not have anything uploaded. The render for me was correct but when I added fallback true in getStaticPaths is when there were problems of rendering [business]/index.ts where business was undefined. Setting it to false solved the errors, but I would like to activate the fallback.

Regards
